### PR TITLE
fix some ui api doc

### PR DESCRIPTION
**Pull request by LinYunMo**
Re: https://github.com/cocos/cocos-engine/issues/10949

### CODE REVIEW

File: cocos/2d/framework/ui-transform.ts [hunk 1]
```diff
@@ -527,9 +529,12 @@ export class UITransform extends Component {
      * 将一个 UI 节点世界坐标系下点转换到另一个 UI 节点 (局部) 空间坐标系，这个坐标系以锚点为原点。
      * 非 UI 节点转换到 UI 节点(局部) 空间坐标系，请走 Camera 的 `convertToUINode`。
      *
-     * @param worldPoint - 世界坐标点。
-     * @param out - 转换后坐标。
-     * @returns - 返回与目标节点的相对位置。
+     * @param worldPoint @en World coordinate point.
```
Comment (SantyWang) [inline cocos/2d/framework/ui-transform.ts]: point in world space

File: cocos/2d/framework/ui-transform.ts [hunk 2]
```diff
@@ -527,9 +529,12 @@ export class UITransform extends Component {
...
+     *                   @zh 世界坐标点。
+     * @param out @en Coordinates after conversion.
```
Comment (SantyWang) [inline cocos/2d/framework/ui-transform.ts]: point in local space

File: cocos/2d/framework/ui-transform.ts [hunk 3]
```diff
@@ -552,9 +557,12 @@ export class UITransform extends Component {
      * @zh
      * 将距当前节点坐标系下的一个点转换到世界坐标系。
      *
-     * @param nodePoint - 节点坐标。
-     * @param out - 转换后坐标。
-     * @returns - 返回 UI 世界坐标系。
+     * @param nodePoint @en Node coordinates.
```
Comment (SantyWang) [inline cocos/2d/framework/ui-transform.ts]: point in local space

File: cocos/2d/renderer/batcher-2d.ts [hunk 4]
```diff
@@ -578,8 +581,10 @@ export class Batcher2D implements IBatcher {
      * @zh
      * 强行修改当前批次数据并合并。
      *
-     * @param material - 当前批次的材质。
-     * @param sprite - 当前批次的精灵帧。
+     * @param material @en The material of the current batch.
+     *                 @zh 当前批次的材质。
+     * @param sprite @en The current batch of sprite frames.
```
Comment (SantyWang) [inline cocos/2d/renderer/batcher-2d.ts]: sprite frame of current batch

File: cocos/2d/renderer/batcher-2d.ts [hunk 5]
```diff
@@ -148,7 +148,8 @@ export class Batcher2D implements IBatcher {
      * @zh
      * 添加屏幕组件管理。
      *
-     * @param comp - 屏幕组件。
+     * @param comp @en The Screen Component.
```
Comment (SantyWang) [inline cocos/2d/renderer/batcher-2d.ts]: the render root of 2d

**Review by SantyWang — APPROVED** [review]

**Comment by pandamicro** [comment]
https://github.com/cocos/cocos-engine/issues/10840

**Pull request merged**